cmake/external-libraries.cmake

```diff
@@ -54,6 +54,8 @@ else()
     add_link_options(-L${kphp-timelib_SOURCE_DIR}/objs)
 endif()
 
+add_compile_options(-Wno-redundant-move)
```

Comment:
Let's move the `kphp-timelib` installation to the top, to use the flag to build other libraries. Also let's add an explanatory comment why this option is added here

Reply:
Ok, done